src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs

```diff
@@ -1,29 +1,27 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Options;
-using OrchardCore.Environment.Shell;
 using OrchardCore.ResourceManagement;
 using OrchardCore.Settings;
 
 namespace OrchardCore.Resources
 {
     public class ResourceManagementOptionsConfiguration : IConfigureOptions<ResourceManagementOptions>
     {
-        private readonly ISiteService _siteService;
+        private readonly ResourceOptions _resourceOptions;
         private readonly IHostEnvironment _env;
         private readonly IHttpContextAccessor _httpContextAccessor;
-        private readonly string _pathBase;
-
-        private const string cloudflareUrl = "https://cdnjs.cloudflare.com/ajax/libs/";
+        private readonly PathString _pathBase;
         // Versions
         private const string codeMirrorVersion = "5.65.4";
         private const string monacoEditorVersion = "0.33.0";
         // URLs
+        private const string cloudflareUrl = "https://cdnjs.cloudflare.com/ajax/libs/";
         private const string codeMirrorUrl = cloudflareUrl + "codemirror/" + codeMirrorVersion + "/";
 
-        public ResourceManagementOptionsConfiguration(ISiteService siteService, IHostEnvironment env, IHttpContextAccessor httpContextAccessor, ShellSettings shellSettings)
+        public ResourceManagementOptionsConfiguration(IOptions<ResourceOptions> resourceOptions, IHostEnvironment env, IHttpContextAccessor httpContextAccessor)
         {
-            _siteService = siteService;
+            _resourceOptions = resourceOptions.Value;
             _env = env;
             _httpContextAccessor = httpContextAccessor;
             _pathBase = _httpContextAccessor.HttpContext.Request.PathBase;
```

Comment:
If you resolve pathBase here then no need to keep the private field `_httpContextAccessor`, if not used below